oidc/identity.go

```diff
@@ -8,10 +8,11 @@ import (
 )
 
 type Identity struct {
-	ID        string
-	Name      string
-	Email     string
-	ExpiresAt time.Time
+	ID              string
+	Name            string
+	Email           string
+	ExpiresAt       time.Time
+	AdditonalClaims jose.Claims
 }
 
 func IdentityFromClaims(claims jose.Claims) (*Identity, error) {
```

Comment:
I'd assume that `IdentityFromClaims` would also fill the `AdditionalClaims` field